qa/automation/CareerPortal.Tests.Scenarios/Career Portal/Send your CV.feature

```diff
@@ -0,0 +1,170 @@
+Feature: "Send us your CV" form
+
```

Comment:
I think you did a great job 
It was a pleasure to check your work :)

Reply:
Thank you! :)